networks/mainnet.json

```diff
@@ -13,7 +13,7 @@
         "Pair_ETH_USDC": "0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc",
         "GovernorAlpha": "0xc0dA01a04C3f3E0be433606045bB7017A7323E38",
         "cDAI": "0x5d3a536E4D6DbD6114cc1Ead35777bAB948E3643",
-        "CompoundLens": "0xd513d22422a3062Bd342Ae374b4b9c20E0a9a074",
+        "CompoundLens": "0xA6c8D1c55951e8AC44a0EaA959Be5Fd21cc07531",
```

Comment:
We do have it here: https://github.com/compound-finance/compound-config/blob/65fb0c356064581d820d967f6a8be1d5183c2b53/networks/mainnet.json#L18